crates/wast-util/src/lib.rs

```diff
@@ -252,10 +286,72 @@ impl Compiler {
                     return true;
                 }
             }
+
+            // Pulley is just getting started, it implements almost no proposals
+            // yet.
+            Compiler::Pulley => {
+                let TestConfig {
+                    memory64,
+                    custom_page_sizes,
+                    multi_memory,
+                    threads,
+                    gc,
+                    function_references,
+                    relaxed_simd,
+                    reference_types,
+                    tail_call,
+                    extended_const,
+                    wide_arithmetic,
+                    simd,
+                    gc_types,
+                    hogs_memory: _,
+                    nan_canonicalization: _,
+                    component_model_more_flags: _,
+                } = *config;
+
+                // Unsupported proposals
+                if memory64.unwrap_or(false)
+                    || custom_page_sizes.unwrap_or(false)
+                    || multi_memory.unwrap_or(false)
+                    || threads.unwrap_or(false)
+                    || gc.unwrap_or(false)
+                    || function_references.unwrap_or(false)
+                    || relaxed_simd.unwrap_or(false)
+                    || reference_types.unwrap_or(false)
+                    || tail_call.unwrap_or(false)
+                    || extended_const.unwrap_or(false)
+                    || wide_arithmetic.unwrap_or(false)
+                    || simd.unwrap_or(false)
+                    || gc_types.unwrap_or(false)
```

Comment:
Maybe we should

```rust
impl TestConfit {
    fn custom_page_sizes(&self) -> bool {
        self.custom_page_sizes.unwrap_or(false)
    }

    // etc...
}
```

if we are doing this dance every time we access these fields